test/extended/security/supplemental_groups.go

```diff
@@ -72,24 +69,16 @@ var _ = g.Describe("[security] supplemental groups", func() {
 			o.Expect(groupAdd).ToNot(o.BeEmpty(), fmt.Sprintf("groupAdd on host config was %v", groupAdd))
 
 			g.By("ensuring the groups are set")
-			o.Expect(configHasGroup(fsGroup, dockerContainer.HostConfig)).To(o.Equal(true), fmt.Sprintf("fsGroup should exist on host config: %v", groupAdd))
-			o.Expect(configHasGroup(supGroup, dockerContainer.HostConfig)).To(o.Equal(true), fmt.Sprintf("supGroup should exist on host config: %v", groupAdd))
+			group := strconv.FormatInt(fsGroup, 10)
+			o.Expect(dockerContainer.HostConfig.GroupAdd).To(o.ContainElement(group), fmt.Sprintf("fsGroup %v should exist on host config: %v", fsGroup, groupAdd))
```

Comment:
the message vs the expect looks a little weird since it used the `groupAdd` in the message and `dockerContainer.HostConfig.GroupAdd` in the compare.  Granted they're the same thing, it just reads weird to me now.  Suggest using the same var in the expect and in the `fmt.Sprintf`.